ui/app/components/job-status/allocation-status-block.hbs

```diff
@@ -13,6 +13,9 @@
           @label="View allocation"
         >
           {{#if (and (eq @status "running") (not @steady))}}
+            {{#if (eq @canary "canary")}}
+              <span class="alloc-canary-indicator" />
+            {{/if}}
```

Comment:
Side-effect: we weren't showing the canary indicator for running, non-steady, ungrouped allocs. Now we are.